ports/qemu-arm/Makefile

```diff
@@ -1,6 +1,8 @@
 include ../../py/mkenv.mk
 -include mpconfigport.mk
 
+FROZEN_MPY_DIR ?= modules
```

Comment:
Will standard (non-test) qemu-arm build with this, given the directory doesn't exist?  Probably this line should be deleted.